api/profile/profileRouter.js

```diff
@@ -287,4 +288,22 @@ router.put(
   }
 );
 
+//get match mentor by profile_id
+router.get('/match/:id', authRequired, (req, res, next) => {
+  axios
+    .post(`${process.env.DS_API_URL}match/${req.params.id}/?n_matches=5`)
```

Comment:
I believe the DS baseURL changed in our product resource docs, in which case there there may need to be a / before match on line 294? Note, your .env file will also need to be updated in order for this to work locally.

Reply:
thanks Alex, updated added a / before match.